tests/layer_tests/ovc_python_api_tests/test_complex_params.py

```diff
@@ -70,13 +79,27 @@ def create_tf_model_single_input_output(tmp_dir):
         # save model to .pb and return path to the model
         return save_to_pb(tf_net, tmp_dir)
 
-    test_data = [
+    paddle_test_data_set_1 = [
```

Comment:
I would suggest to split these tests into separate files by FWs

Reply:
Done.